### PR TITLE
Update entrance field check in VenuePage

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Tries to resolve:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1088

by updating the check of field `entrance` from `isDefined()` to `Array.isArray(entrance) && entrance.length > 0`

Based on comment https://github.com/sparkletown/internal-sparkle-issues/issues/1088#issuecomment-913609828

### CODE REVIEW

File: src/pages/VenuePage/VenuePage.tsx [hunk 1]
```diff
@@ -209,18 +208,17 @@ export const VenuePage: React.FC = () => {
   // if (isAccessDenied) {
   //   return <AccessDeniedModal venueId={venueId} venueName={venue.name} />;
   // }
+  const { entrance, template, hasPaidEvents } = venue;
+  const { enteredVenueIds } = profile;
+
+  const hasEntrance = Array.isArray(entrance) && entrance.length > 0;
```
Comment (goran-peoski-work) [inline src/pages/VenuePage/VenuePage.tsx:214]: Entrance step will now only work if the field in DB is an array with at least one element.

**Review by goran-peoski-work — COMMENTED** [review]
Initial PR comments

**Review by mike-lvov — APPROVED** [review]
LGTM

**Comment by whoaitsaimz** [comment]
Did not fix issue. But we know the source of the issue now (an empty array in database), thanks for the detective work, team.

**Comment by goran-peoski-work** [comment]
@whoaitsaimz You're right about the first part. This  PR didn't fix the cause, it's just prevention if we get in the same situation. It's a check when reading invalid data.

What we didn't discover is how the invalid data (empty array) was created. Might be manual DB change, might be some code of a form somewhere. Hopefully we'd not need to after this PR merge.

P.S. All the detective credit should go to @mike-lvov  who did 💯💪 of it.

**Comment by goran-peoski-work** [comment]
@whoaitsaimz Just read https://github.com/sparkletown/internal-sparkle-issues/issues/1088#issuecomment-918896384

I stand corrected. I can safely say about 80% or more of the detective credit should have gone to you. Sorry @mike-lvov .

Agree with the conclusion, we shouldn't do more PRs for this problem.

**Pull request merged**